### ISSUE TITLE
error in image width of the jem teaser module 

### ISSUE THREAD

**Issue by Heklaterriol** [Heklaterriol/JEM-Project#174]
in `modules/mod_jem_teaser/tmpl/responsive/default.php`

there is on line 71

```
  #jemmoduleteaser .jem-eventimg-teaser {
    width: <?php echo $imagewidth; ?>;
  }
```

which catches also the text block and leads to a very small column:
![teaser](https://user-images.githubusercontent.com/28540581/228094266-c0243268-2e3a-4830-932a-ae0caae294e5.png)

The text block should remain full with, ¿no es así?

**Comment by mckillo** [comment]
![JEM - Module TEaser- description responsive](https://user-images.githubusercontent.com/2765316/228218985-f7f3c4e9-4012-4a9d-862b-9eb8fa6422f4.jpg)

@Heklaterriol Solved, but it was a issue about description width. Tell me if it's right. Check it.
See commit 9f00fbbe8b5178682ec66badbbff7333a7a03492

**Comment by jojo12** [comment]
a question: did we forget the text in mod_jem_teaser?

**Comment by jojo12** [comment]
Ah, I see, I get eventdescription only, when position is in the content area (main top /main bottom

**Comment by mckillo** [comment]
I agree. Now, you have a new option in module to show description or no, independent of other options.

![JEM - Module Teaser - New option - Show description](https://user-images.githubusercontent.com/2765316/229652607-aa036533-2274-45c7-8d08-8f02f39a3eb3.jpg)

See commit 78005abfbf6340bf8cc8812354c7223af45a72e5
I have discovered a old issue about 'error save setting in css_backend_usecustom', in this case in migration of version from 2.3.5 with J3. For it in sql script fixed the solution #60 (update 2.3.16.sql). 

If you want to test it, download the new 2.3.16 zip files updated with last commit from:
 https://github.com/mckillo/JEM-Project/blob/JEM-4beta2/pkg_jem_v2.3.16_updated.zip

**Comment by jojo12** [comment]
it's good to have this button.

**Comment by mckillo** [comment]
You have it, See above commit 78005abfbf6340bf8cc8812354c7223af45a72e5ee
